### PR TITLE
Add badges to Readme.md

### LINKED ISSUE #17: Badges
Badges from:

- https://readthedocs.org/projects/lira/
- https://codecov.io/gh/pythonecuador/lira/
- https://github.com/pythonecuador/lira/actions
- https://pypi.org/project/lira/

### PR DESCRIPTION
**Pull request by rpalaciosg**
Add the following Badges:
- documentation
- coverages
- test
- pypi

image

Closes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # lira
 
+[![Documentation Status](https://readthedocs.org/projects/lira/badge/?version=latest)](https://lira.python.ec/en/latest/?badge=latest)
+[![codecov.io](https://codecov.io/github/pythonecuador/lira/coverage.svg?branch=master)](https://codecov.io/github/pythonecuador/lira?branch=master)
+![tests](https://github.com/pythonecuador/lira/workflows/tests/badge.svg)
+[![PyPI version](https://badge.fury.io/py/lira.svg)](https://badge.fury.io/py/lira)
```
Comment (stsewd) [inline README.md:6]: This should link to https://pypi.org/project/lira/

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # lira
 
+[![Documentation Status](https://readthedocs.org/projects/lira/badge/?version=latest)](https://lira.python.ec/en/latest/?badge=latest)
+[![codecov.io](https://codecov.io/github/pythonecuador/lira/coverage.svg?branch=master)](https://codecov.io/github/pythonecuador/lira?branch=master)
+![tests](https://github.com/pythonecuador/lira/workflows/tests/badge.svg)
```
Comment (stsewd) [inline README.md:5]: This should link to https://github.com/pythonecuador/lira/actions?query=workflow%3Atests+branch%3Amaster

Comment (rpalaciosg) [reply README.md]: Ok, update the links for test and pypi badges.

**Review by stsewd — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (rpalaciosg) [reply README.md]: Ok, update the link.

**Pull request merged**